src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,51 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Design System open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import * as RadioPrimitive from '@radix-ui/react-radio-group'
+import {
+  forwardRef,
+  type ElementRef,
+  type ComponentPropsWithoutRef,
+} from 'react'
+import { cn } from '../../utils/className'
+
+// entire list of choices
```

Comment:
These kind of comments should be using JSDoc notation. 

```suggestion
/**
 * entire list of choices
 * */
```

JSDoc allows writing documentation, it's a standard widely used. Editors rely on it to display hints in your IDE. Probably they'd work with regular comment as well, but let's keep it as JSDoc format.

Example JSDoc usage: [explaining maxToDisplay property of Breadcrumbs](https://github.com/StanfordBDHG/speziwebdesignsystem/blob/0fed267b7d111bd5d8616a0a00e8067c09094717/src/components/Breadcrumb/Breadcrumb.tsx#L165-L170)

Reply:
When it comes to the content of this comment what do you think about something like `Container for Radio inputs`?